src/main/java/core/basesyntax/dao/fileWriterImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.dao;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class fileWriterImpl implements fileWriter {
```

Comment:
The class name `fileWriterImpl` does not follow Java naming conventions. Class names should be in PascalCase, such as `FileWriterImpl`.